### PR TITLE
test: 💍 credential store CRUDL tests

### PR DESCRIPTION
**Pull request by lisbet-alvarez**
✅ Closes: ICU-5329

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-5329)

## Description
Acceptance tests revised to test both static and vault types of credential stores.

### CODE REVIEW

File: ui/admin/tests/acceptance/credential-store/update-test.js [hunk 1]
```diff
@@ -38,54 +37,74 @@ module('Acceptance | credential-stores | update', function (hooks) {
...
-    instances.credentialStore.authorized_actions =
-      instances.credentialStore.authorized_actions.filter(
+    instances.staticCredentialStore.authorized_actions =
+      instances.staticCredentialStore.authorized_actions.filter(
         (item) => item !== 'update'
       );
-    await visit(urls.credentialStore);
+    await visit(urls.staticCredentialStore);
     assert.notOk(find('.rose-layout-page-actions .rose-button-secondary'));
   });
```
Comment (cameronperera) [inline ui/admin/tests/acceptance/credential-store/update-test.js:96]: Do we need a matching test for Vault Credential stores here?

**Review by cameronperera — CHANGES_REQUESTED** [review]
Overall looks good. In the update-test we have test for the static Credential Stores but not for Vault types. We should add those in as well.

File: ui/admin/tests/acceptance/credential-store/create-test.js [hunk 2]
```diff
@@ -38,29 +39,44 @@ module('Acceptance | credential-stores | create', function (hooks) {
...
   });
 
-  test('Users can create a new credential stores', async function (assert) {
+  test('Users can create a new credential store of default type static', async function (assert) {
     assert.expect(1);
-    const count = getCredentialStoresCount();
+    const count = getStaticCredentialStoresCount();
+    await visit(urls.newCredentialStore);
+    await fillIn('[name="name"]', 'random string');
+    await click('[value="static"]');
```
[the remark this answers is not in the document]
Comment (lisbet-alvarez) [reply ui/admin/tests/acceptance/credential-store/create-test.js:67]: Great Idea!

**Review by cameronperera — APPROVED** [review]
LGTM

**Review by DhariniJeeva — APPROVED** [review]

**Pull request merged**